### ISSUE TITLE
Reports AttributeError: module 'os' has no attribute 'uname' when using Windows release

### ISSUE THREAD

**Issue by 1847905557** [Wenzel/checksec.py#81]
```
checksec.exe 51ed19eacdea43e3bd67217d08eb8a0e

Processing... ────────────────────────────────────────────────────────────────────────────── 1/1 • 100.0%
concurrent.futures.process._RemoteTraceback:
"""
Traceback (most recent call last):
  File "concurrent\futures\process.py", line 175, in _process_worker
  File "checksec\__main__.py", line 51, in checksec_file
  File "checksec\elf.py", line 181, in checksec_state
  File "checksec\elf.py", line 169, in fortified
  File "checksec\elf.py", line 162, in __get_libc
  File "checksec\elf.py", line 59, in __init__
  File "checksec\utils.py", line 29, in find_libc
  File "checksec\utils.py", line 71, in find_library_full
AttributeError: module 'os' has no attribute 'uname'
"""


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "stub.py", line 7, in <module>
  File "checksec\__main__.py", line 115, in entrypoint
  File "checksec\__main__.py", line 89, in main
  File "concurrent\futures\_base.py", line 425, in result
  File "concurrent\futures\_base.py", line 384, in __get_result
AttributeError: module 'os' has no attribute 'uname'
[17784] Failed to execute script stub
```

**Comment by Wenzel** [comment]
Hi @1847905557 and thanks for the bug report.

Indeed, I have Linux specific code when parsing an ELF file, so it won't work on Windows.

I need to refactor the tool a bit to make this work, but I will work on it, it's definitely a bug !

Thanks

**Comment by Wenzel** [comment]
Hi @1847905557 

I believe your bug has been fixed by the last release `v0.5.0`:
https://twitter.com/mtarral/status/1321229882622464003?s=20

(related: https://github.com/Wenzel/checksec.py/pull/88, https://github.com/Wenzel/checksec.py/pull/87, https://github.com/Wenzel/checksec.py/pull/86, https://github.com/Wenzel/checksec.py/pull/85)

Thanks again for the bug report !
Can you confirm it works and close the issue ?

**Comment by 1847905557** [comment]
> 
> 
> Hi @1847905557
> 
> I believe your bug has been fixed by the last release `v0.5.0`:
> https://twitter.com/mtarral/status/1321229882622464003?s=20
> 
> (related: #88, #87, #86, #85)
> 
> Thanks again for the bug report !
> Can you confirm it works and close the issue ?

It works as normal, I think it has been fixed.
Thanks for your fantastic works!
